frontend/packages/ux-editor-v3/src/components/config/Expressions/ExpressionContent/ExpressionPreview/ExpressionPreview.test.tsx

```diff
@@ -8,19 +8,17 @@ import {
 } from '../../../../../testing/expressionMocks';
 import type { ServicesContextProps } from 'app-shared/contexts/ServicesContext';
 import { renderWithMockStore } from '../../../../../testing/mocks';
-import { formDesignerMock } from '../../../../../testing/stateMocks';
 import type { IFormLayouts } from '../../../../../types/global';
-import { layout1NameMock, layoutMock } from '../../../../../testing/layoutMock';
-import { textMock } from '../../../../../../../../testing/mocks/i18nMock';
+import { layout1NameMock, layoutMock, layoutSet1NameMock } from '../../../../../testing/layoutMock';
```

Comment:
Should this be ⬇️?
```suggestion
import { layout1NameMock, layoutMock, layoutSet1NameMock } from '@studio/testing/layoutMock';
```

Reply:
Same here as in [my previous comment](https://github.com/Altinn/altinn-studio/pull/12797#discussion_r1611766890)